plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -166,4 +171,56 @@ public IcebergConfig setProjectionPushdownEnabled(boolean projectionPushdownEnab
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    @NotNull
+    public DataSize getSplitTargetSize()
+    {
+        return splitTargetSize;
+    }
+
+    @Config("iceberg.split-target-size")
+    public IcebergConfig setSplitTargetSize(DataSize splitTargetSize)
+    {
+        this.splitTargetSize = splitTargetSize;
+        return this;
+    }
+
+    @NotNull
+    public DataSize getSplitMetadataTargetSize()
+    {
+        return splitMetadataTargetSize;
+    }
+
+    @Config("iceberg.split-metadata-target-size")
+    public IcebergConfig setSplitMetadataTargetSize(DataSize splitMetadataTargetSize)
+    {
+        this.splitMetadataTargetSize = splitMetadataTargetSize;
+        return this;
+    }
+
+    public int getSplitPlanningLookback()
+    {
+        return splitPlanningLookback;
+    }
+
+    @Config("iceberg.split-planning-lookback")
+    @ConfigDescription("The number of bins considered when trying to pack the next file split into a scan task.")
+    public IcebergConfig setSplitPlanningLookback(int splitPlanningLookback)
```

Comment:
As a user, when would i want to set it?

Reply:
Iceberg using `CombinedScanTask` to pack `FileScanTask` (file split), in order to make tasks more even, during planning scan tasks.

Reply:
> As a user, when would i want to set it?

Iceberg using `CombinedScanTask` to pack `FileScanTask` (file split), in order to make tasks more even, during planning scan tasks. In this procedure, a `bin` is just a `CombinedScanTask`. Iceberg supports specify the target size of the `bin`, so when packing the next file split, Iceberg can look back `splitPlanningLookback` bins to determine to put the split into one of the looked bins or a new bin. In this way to reduce the small tasks, hopefully, but surely, it will sacrifice the performance.

So, this conf is a tradeoff, a smaller value will make it more quick to generate the scan tasks, but maybe with more small tasks; a higher value will reduce the small tasks, but decrease the speed of generating tasks.

Reply:
BTW, it's good to set this property as `1` by default ?  Because Trino will unfold `CombinedScanTask` into `FileScanTask`.